### ISSUE TITLE
empty Mat

### ISSUE THREAD

**Issue by HetNaik997** [diviramon/LiveFeedProcessing#1]
Hi! I am a student working with Allied vision camera doing image processing.
I came across this project and it a wonderful solution to my problem.  I wanted to ask if i can use this code for my project?
I actually have a small doubt about the program.  I used it and i am getting an error which suggests that the cv :: mat is actually empty. I am very new to programming and i don not know how to figure this out. Can you give me some hint about ? 

Thank you very much for working on this project and making it available.

Thank you for your consideration.

**Comment by diviramon** [comment]
Hi! Feel free to use the code. The empty mat issue is most likely because your camera is not sending any frames, or the program cannot connect to it.

**Comment by HetNaik997** [comment]
Hello! thank you very much for responding. I have tried connecting to the camera and it worked. It is showing the camera IP and all the information which means camera is connected but I cannot verify why it is not sending any frames to the system. 

Thank you very much for helping me this far.

**Comment by diviramon** [comment]
Do you get this message?

"Error starting the stream with the camera, ending..."

**Comment by HetNaik997** [comment]
Hi! No actually. I am getting something like " OpenCV (4.5.0) Error: Assertion failed (size.width> 0 && size.height> 0) folder \ opencv-4.5.  I can get from this is that there is no frame allocated for the Mat. 

Do you have any possible fixes for this??

**Comment by diviramon** [comment]
Correct, so the camera is not sending any frames. I would try to run a smaller script and make sure that your camera is compatible with the VimbSDK version that you have installed, and that you can take and display an image.
